include/mbedtls/platform.h

```diff
@@ -219,11 +220,11 @@ int mbedtls_platform_set_snprintf( int (*snprintf_func)( char * s, size_t n,
 extern void (*mbedtls_exit)( int status );
 
 /**
- * \brief   Set your own exit function pointer
+ * \brief   This function sets your own exit function pointer
  *
- * \param exit_func   the exit function implementation
+ * \param exit_func   The exit function implementation.
  *
- * \return              0
+ * \return              0.
```

Comment:
To be uniform, this should be `\c 0 on success`, too.

Reply:
fixed